pages/common/docker-run.md

```diff
@@ -0,0 +1,28 @@
+# docker run
+
+> Run a command in a new Docker container.
+> More information: <https://docs.docker.com/engine/reference/commandline/run/>.
+
+- Run command in a new container from a tagged image:
+
+`docker run {{image:tag}} {{command}}`
+
+- Run command in a new container in background and display its ID:
```

Comment:
To match the `-d`, it might be better to say "in a new container as a background daemon .."